pages/index.js

```diff
@@ -76,17 +95,21 @@ const Game = () => {
   //  Render the Game
   return (
     <div>
-      <MovesProvider>
-        <p className="turn">
-          Who's turn? Player {winner ? `${winner} wins` : player}
-          <button className="new-game" onClick={() => setGame({})}>
-            New game
+      {/* <MovesProvider> */}
```

Comment:
I took out the MovesProvider because you aren't using it.

Reply:
Ah yeah, I'm saving for that for later :) Thanks for the help!